.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
This will install the latest version, right? That means that our builds are not deterministic, but depend on the bundler project releasing new versions. I'm wondering what's worse, automatically upgrading to the latest version or updating the Travis config regularly.

We could do something like:
```
gem install bundler -v "$(tail Gemfile.lock -n 1)"
```
But I'm wondering which bundler versions our production servers are running. Isn't that more important than our dev machines?

Aus production: Bundler version 1.10.6

It's also important to keep backwards compatibility. So having a new version on a dev machine and testing against an older version can be very useful as well.

What are your thoughts on that?